bode/bode/models/task.py

```diff
@@ -16,9 +16,6 @@ class TaskStatus(enum.Enum):
     INDIRECTLY_DONE = "INDIRECTLY_DONE"
     DONE = "DONE"
 
-    def __str__(self):
-        return self.value
```

Comment:
It was needed to cast Postgres-Python Enum value to display just a value. Not sure if these changes without any migrations will work

Reply:
I don't understand what might not work. Could you explain?